pkgs/games/dwarf-fortress/lazy-pack.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenvNoCC, lib, buildEnv, callPackage
+, dwarf-fortress, themes
+  # This package should, at any given time, provide an opinionated "optimal"
+  # DF experience. It's the equivalent of the Lazy Newbie Pack, that is, and
+  # should contain every utility available.
+, enableDFHack ? true
```

Comment:
I might be wrong but I think if you set this to ```stdenv.isLinux``` this will support macOS out of the box. I'll need to test to verify though.

Reply:
Actually I think a few of the wrappers might still be broken.

Reply:
You might have an older version. I noticed I'd used lib.optional wrongly, which broke the legends-browser and dwarf-therapist packages.

Reply:
There's still a few places we assume linux actually but I think we're closer:

https://github.com/NixOS/nixpkgs/blob/master/pkgs/games/dwarf-fortress/dwarf-therapist/wrapper.nix#L6

But I think it's closer. I tried to get dfhack working on macOS but it was more difficult than I thought.